libs/remix-ui/terminal/src/lib/remix-ui-terminal.tsx

```diff
@@ -78,6 +74,33 @@ export const RemixUiTerminal = (props: RemixUiTerminalProps) => {
   // terminal dragable
   const leftRef = useRef(null)
   const panelRef = useRef(null)
+  const terminalMenu = useRef(null)
+
+  const terminalMenuOffsetHeight = (terminalMenu.current && terminalMenu.current.offsetHeight) || 35
+  const terminalDefaultPosition = config.get('terminal-top-offset')
+
+  const {
+    isOpen,
+    isDragging,
+    terminalPosition,
+    handleDraggingStart,
+    handleToggleTerminal
+  } = useDragTerminal(terminalMenuOffsetHeight, terminalDefaultPosition)
+
+  // Check open state
+  useEffect(() => {
+    const resizeValue = isOpen ? [config.get('terminal-top-offset')] : []
+    event.trigger('resize', resizeValue)
+  }, [isOpen])
+
+  useEffect(() => {
+    leftRef.current.style.backgroundColor = isDragging ? '#007AA6' : ''
+    leftRef.current.style.border = isDragging ? '2px solid #007AA6' : ''
+  }, [isDragging])
```

Comment:
I don't see a need for this useEffect. The logic above can be contained in the style of the element.

Reply:
can you please use theme variables/classes instead of hardcoded values for color? we have different themes, it should work for all of them

Reply:
That logic I left it as it was, anyway I already sent the changes.

There are other hardcoded colors in the code, maybe I can verify that they change according to the different themes and send a pr if necessary.

Thanks for your reviews!